src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private final double side;
+
+    public Square(String color, double side) {
```

Comment:
Avoid passing expressions directly into the constructor. For better readability and maintainability, assign the value of `side` to a variable before passing it to the constructor.